data/brands/shop/department_store.json

```
@@ -218,6 +218,18 @@
         "shop": "department_store"
       }
     },
+    {
+      "displayName": "Burlington Coat Factory",
+      "id": "burlingtoncoatfactory-0a21d9",
+      "locationSet": {"include": ["pr", "us"]},
+      "tags": {
+        "brand": "Burlington Coat Factory",
+        "brand:wikidata": "Q4999220",
+        "brand:wikipedia": "en:Burlington (department store)",
+        "name": "Burlington Coat Factory",
+        "shop": "clothes"
```

If you want to tag this as `shop=clothes` this should be moved to the existing `shop/clothes` category
```suggestion
        "shop": "department_store"
```

---

Another note on Burlington. They're just using the name "Burlington" now. We should set "Burlington Coat Factory" as an alt_name.